### DISCUSSION TITLE
can i change some manifest value for a package and have it persist updates?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [ScoopInstaller/Scoop#5848] [Help and Questions] [answered]
hi! thanks a lot for such wonderful offering.

i want to ask if i
can i change some manifest value for a package and have it persist updates?

case in hand, i want to change/remove the `env_add_path` value for `vscode` as i don't want to add it to path. 

```console
$ # XARGS: -o: --open-tty, -i: -I {}: --replace={}
$ # The '> ' prompt is manually added here - just to help the parser. 
$ # Idk if it's possible to specify this in xargs

$ xargs -oi scoop cat {} | jq '{extract_dir, persist, env_add_path, env_set}'
> vscodium
{
  "extract_dir": null,
  "persist": "data",
  "env_add_path": "bin",
  "env_set": null
}
> vscode
{
  "extract_dir": null,
  "persist": "data",
  "env_add_path": "bin",
  "env_set": null
}

```

<details>
<summary>
blabbering - please ignore
</summary>

reason being:
`code` name sucks. i don't want allow microsoft to litter my path with cheap names.
besides, i don't want even "vscode" to be accidently openable from CLI, say by someone else giving my pc a whirl.
all aside, if being honest... it's just my way to feel good about blocking it. it's useless way, but it makes me relieved.

</details>

_A portion of it originally posted by @me in https://github.com/ScoopInstaller/Scoop/discussions/5843#discussioncomment-8902760_

**Comment by niheaven** [comment]
One of the advantages of scoop is that you can create your own manifest and store it in your own bucket. Just do it.

**Reply by goyalyashpal** [reply]
hehe. 'am not gonna bother with that for exising manifests for now. 
reducing hassles is the reason i invested in migration to scoop. 

but thanks for the invitation 😄

**Comment by goyalyashpal** [comment, marked as answer]
> One of the advantages of scoop is that you can create your own manifest and store it in your own bucket. Just do it.
>
> \- @ niheaven _at https://github.com/ScoopInstaller/Scoop/discussions/5848#discussioncomment-8910660_

yeah, if scoop allowed overriding one package to create another package (like nixpkgs allows) then it would have been a different matter.

as that way, i wont have to keep track of other changes / fixes in the base of package.

_here: package means the metadata consumable by the package manager i.e. scoop_

----
; always feel that scoop in _MANY_ sense seems to be inspired by nixpkgs.  and this could be yet another thing it should take inspiration from nixpkgs lol.

but this is not one way, as been using nixos for almost 10-11 months, i can clearly see the places where nixpkgs stould take inspiration from scoop.

----
